app/lib/pdf/styles/base.rb

```diff
@@ -55,15 +55,45 @@ def freeze_style_hash(struct = styles)
       end
     end
 
-    private
+    class FontLiberation
+      FILE_REGULAR = "LiberationSans-Regular.ttf"
+      FILE_BOLD = "LiberationSans-Bold.ttf"
+      FAMILY = "Liberation Sans"
+
+      private_constant :FAMILY, :FILE_BOLD, :FILE_REGULAR
+
+      class << self
+        def family
+          FAMILY
+        end
 
-    def liberation_dir
-      %w[
-      /usr/share/fonts/liberation
-      /usr/share/fonts/liberation-sans
-      ].find do |path|
-        File.directory? path
+        def bold
```

Comment:
I'd prefer to have it as `bold_path` and `regular_path`, so it's clearer what kind of value to expect.